api/openshift.deploy.yml

```diff
@@ -325,7 +303,7 @@ objects:
               containers:
                 - name: ${NAME}-batch
                   # Full image name required due to bug https://bugzilla.redhat.com/show_bug.cgi?id=2000216
-                  image: image-registry.apps.silver.devops.gov.bc.ca/${OC_NAMESPACE}/${NAME}-${ZONE}-${COMPONENT}:${ZONE}-api
+                  image: ${REGISTRY}/${ORG}/nr-fom/${COMPONENT}:${ZONE}
```

Comment:
Is this the Github repo now?

Reply:
Yup!  Packages viewable [here](https://github.com/orgs/bcgov/packages?repo_name=nr-fom).  :)